deeplearning4j/deeplearning4j-core/src/test/java/org/deeplearning4j/nn/conf/ComputationGraphConfigurationTest.java

```diff
@@ -217,6 +217,32 @@ public void testInvalidConfigurations() {
             //OK - exception is good
             //e.printStackTrace();
         }
+
+        //Test: input != inputType count mismatch
+        try {
+             new NeuralNetConfiguration.Builder().graphBuilder().addInputs("input1", "input2")
```

Comment:
This configuration is almost the same as `testJSONWithGraphNodes()` but likely could be trimmed to be shorter?